tests/framework/validators/EmailValidatorTest.php

```diff
@@ -26,8 +26,10 @@ public function testValidateValue()
         $this->assertFalse($validator->validate('Carsten Brandt <mail@cebe.cc>'));
         $this->assertFalse($validator->validate('"Carsten Brandt" <mail@cebe.cc>'));
         $this->assertFalse($validator->validate('<mail@cebe.cc>'));
-        $this->assertFalse($validator->validate('info@örtliches.de'));
-        $this->assertFalse($validator->validate('sam@рмкреатиф.ru'));
+        $this->assertTrue($validator->validate('info@örtliches.de'));
+        $this->assertTrue($validator->validate('sam@рмкреатиф.ru'));
```

Comment:
It's not a valid email according to http://sphinx.mythic-beasts.com/~pdw/cgi-bin/emailvalidate which is claimed to be RFC-compatible.